### PR TITLE
Add full writeups for first few sections

### PR DESCRIPTION
**Pull request by jgeaso1266**
## Key Changes
- adapted the bullet points in the first few sections into paragraph form, and edited based on the needs of the MVP as described in lecture

### CODE REVIEW

File: template/1-overview.md [hunk 1]
```diff
@@ -1,18 +1,9 @@
 # Overview
 
-*Maximum: 2 pages*
+Event Radar is an innovative app designed to effortlessly facilitate connections among individuals by streamlining event discovery, registration, and networking all into one place. It allows users to discover and register for upcoming events in their local area, enabling them to connect seamlessly with friends and new acquaintances they meet during the event. Event hosts can easily create and manage events, while also leveraging the app’s unique QR code system to scan tickets and enhance attendee engagement. By focusing on building a vibrant local community, Event Radar aims to revolutionize the way people interact and form connections in person.
```
Comment (Nath-Assa) [inline template/1-overview.md]: Question:
Here: "Event hosts can easily create and manage events, while also leveraging the app’s unique QR code system to scan tickets and enhance attendee engagement."
What do you mean by enhance attendee engagement?

File: template/1-overview.md [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
 
-*What is this project about?*
-- Event Radar is an app that helps to _easily_ facilitate connections with others
-- it allows users to discover and register for upcoming events in their area, and easily connect with friends they meet
-- it also allows event hosts to easily create and manage events, and even scan tickets for event goers
-- all of this is done through one all-purpose, unique QR code
-- emphasize building a local community
+The primary purpose of Event Radar is to provide a one-stop solution for finding events, connecting with new friends, messaging people, and even hosting events. It eliminates the barriers to creating meaningful connections after events by using unique QR codes, allowing users to easily retrieve and remember contact information. This approach not only simplifies the process of networking but also encourages users to make live, in-person connections, fostering a stronger sense of community and engagement.
```
Comment (Nath-Assa) [inline template/1-overview.md]: Suggestion: Here I'd emphasize the fact that scanning QR code is the only way to add friends.
Here's a sentence you might want to add: "By making QR code scanning the sole method for adding contacts, Event Radar strongly encourages users to engage directly, enhancing the community feel and fostering more meaningful connections."

File: template/1-overview.md [hunk 3]
```diff
@@ -1,18 +1,9 @@
...
 
-*Include a 3-sentence (max) elevator pitch.*  
-Event Radar is a versatile app that simplifies event discovery, connection-making, and hosting. Find nearby events, register effortlessly, and connect instantly with new friends—all through one unique QR code. Whether you're attending or hosting, Event Radar is the ultimate tool for social networking and event coordination, streamlining connections before, during, and after any gathering.
+In summary, Event Radar is a versatile app that simplifies event discovery, connection-making, and hosting. Find nearby events, register effortlessly, and connect einstantly with new friends—all through one unique QR code. Whether you're attending or hosting, Event Radar is the ultimat tool for social networking and event coordination, streamlining connections before, during, and after any gathering.
```
Comment (Nath-Assa) [inline template/1-overview.md]: Typos: Should be "connect instantly" and "ultimate tool"

Comment (Nath-Assa) [inline template/1-overview.md]: Consider this version which I believe is more impactful:
In summary, Event Radar is a versatile app that simplifies discovering and hosting events while enhancing social connections. Find and register for nearby events effortlessly, and connect instantly with new friends—all through a unique QR code. Whether you're attending or organizing, Event Radar is the ultimate tool for social networking and event coordination, streamlining connections before, during, and after any gathering.

File: template/2-history.md [hunk 4]
```diff
@@ -2,42 +2,6 @@
 
 *Maximum 1 page*
 
-*Describe the V1 POC as planned by Sprint10.*
-- 4 epics:
-    - Epic 1: Allowing Users to Discover and Join Events
-    - Epic 2: Allowing Users to Interact With Others
-    - Epic 3: Allow Event Hosts to Easily Create Events
-    - Epic 4: Allow Events Hosts to Communicate with Participants Easily
+The V1 PoC of Event Radar, as planned by Sprint 10, includes essential features that demonstrate the core functionalities of the app (see more in the MVP section). Users can browse and see upcoming events, engage in in-app private messaging, and utilize QR code generation and scanning for seamless connections. Users can also personalize their experience with editable profiles, providing a foundation for building a local community and facilitating meaningful connections. Additionally, event hosts have the ability to create and edit events, ensuring effective event management.
```
Comment (Nath-Assa) [inline template/2-history.md]: Suggestion: 
Change this sentence: " Users can also personalize their experience with editable profiles, providing a foundation for building a local community and facilitating meaningful connections."
to:
"Users can also personalize their experience by customizing their profiles, enabling them to connect more effectively with others and express their identity."

**Review by Nath-Assa — COMMENTED** [review]
Great work!
Mostly nits, but I've run the text through Grammarly and found some errors. To streamline the process, I suggest using this tool to identify and correct any remaining issues, rather than me providing all the edits in the PR review.

File: template/1-overview.md (hunk 1 above)
Comment (jgeaso1266) [reply template/1-overview.md]: This is referencing the ability for hosts to send messages to event attendees. It is bad wording and can be changed.

File: template/1-overview.md
Comment (Nath-Assa) [inline template/1-overview.md:1]: I have run the overview.md file through Grammarly and updated it to a near-perfect version. Please review it and let me know what you think:
Event Radar is an innovative app designed to effortlessly facilitate connections among individuals by streamlining event discovery, registration, and networking in one place. It allows users to discover and register for upcoming events in their local area, enabling them to connect seamlessly with friends and new acquaintances they meet during the event. Event hosts can easily create and manage events while leveraging the app's unique QR code system to scan tickets and interact with event attendees. By focusing on building a vibrant local community, Event Radar aims to revolutionize the way people interact and form connections in person.

Event Radar is all about fostering meaningful connections. It's a one-stop solution for finding events, connecting with new friends, messaging people, and hosting events. It's designed to eliminate barriers to creating meaningful connections after events. With the app's QR code scanning feature, you can easily retrieve and remember contact information, encouraging direct engagement with others. This enhances the sense of community and fosters more meaningful connections. This approach simplifies the networking process and promotes live, in-person interactions, fostering a stronger sense of community and engagement.

Event Radar is a comprehensive platform that's got all your event needs covered. It seamlessly integrates various functionalities into a single platform. You can effortlessly discover events based on your interests and location, register with ease, and instantly connect with others by scanning a simple QR code. For event hosts, the app provides robust tools to manage events efficiently, from ticketing to attendee tracking. This comprehensive approach ensures that both attendees and hosts have an enhanced, streamlined experience, promoting more dynamic and interactive events.

Event Radar is a versatile app that simplifies event discovery, connection-making, and hosting. Find and register for nearby events effortlessly and connect instantly with new friends—all through one unique QR code. Whether attending or hosting, Event Radar is the ultimate tool for social networking and event coordination, streamlining connections before, during, and after any gathering.

File: template/2-history.md
Comment (Nath-Assa) [inline template/2-history.md:1]: Here’s the version with all grammatical and syntax corrections applied:
The V1 PoC of Event Radar, as planned by Sprint 10, includes essential features that demonstrate the app's core functionalities (see more in the MVP section). Users can browse upcoming events, engage in in-app private messaging, and utilize QR code generation and scanning for seamless connections. Users can also personalize their experience by customizing their profiles, enabling them to connect more effectively with others and build a strong community. Additionally, event hosts can create and edit events, ensuring effective event management.
Throughout the development of the PoC, our team faced several learning curves, particularly with the Scrum process. Initially, we encountered challenges with managing large pull requests (PRs) and task clarity, which we addressed by refining task definitions on the Scrum board and limiting the PR scope, leading to fewer dependencies between tasks. Furthermore, the ambitious nature of the app required us to make difficult prioritization decisions due to limited time. Despite these challenges, we successfully laid the groundwork for the MVP. However, further development is needed to implement additional user flows and define features such as group chats, in-event geo-location, and event history tracking to bridge from PoC to MVP fully.

File: template/3-analysis.md
Comment (Nath-Assa) [inline template/3-analysis.md:1]: Perfect!

File: template/4-value_proposition.md
Comment (Nath-Assa) [inline template/4-value_proposition.md:1]: Here's the updated version after minor fixes: 
Event Radar is a one-stop event discovery, registration, and connection-making platform. It simplifies networking by enabling instant friend connections through QR code scanning and in-app messaging and streamlines event hosting with efficient ticket scanning and attendee management tools.
Our product is defensible as it solves many user pain points in one concise app. The unique QR code-based networking feature promotes genuine, physical networking, setting Event Radar apart from traditional event apps. We offer comprehensive event management tools catering to attendees and event hosts, focusing on enhancing post-event connections and creating a sustainable user base.
Finally, Event Radar stays relevant as it meets the growing demand for seamless event experiences local to users in a digital age. It addresses the need for effective networking and relationship-building beyond event attendance, aligning with trends in social interaction and professional networking, especially in urban and academic environments.

**Review by Nath-Assa — APPROVED** [review]
Great work! I've provided some final suggestions. If you'd like to pinpoint the changes, feel free to use a text comparison tool.

File: template/2-history.md [hunk 5]
```diff
@@ -2,42 +2,6 @@
 
 *Maximum 1 page*
 
-*Describe the V1 POC as planned by Sprint10.*
-- 4 epics:
-    - Epic 1: Allowing Users to Discover and Join Events
-    - Epic 2: Allowing Users to Interact With Others
-    - Epic 3: Allow Event Hosts to Easily Create Events
-    - Epic 4: Allow Events Hosts to Communicate with Participants Easily
+The V1 PoC of Event Radar, as planned by Sprint 10, includes essential features that demonstrate the core functionalities of the app (see more in the MVP section). Users can browse upcoming events, engage in in-app private messaging, and utilize QR code generation and scanning for seamless connections. Users can also personalize their experience by customizing their profiles, enabling them to connect more effectively with others and build a strong community. Additionally, event hosts have the ability to create and edit events, ensuring effective event management.
```
Comment (christinehahaa) [inline template/2-history.md]: Add join events:
```suggestion
The V1 PoC of Event Radar, as planned by Sprint 10, includes essential features that demonstrate the core functionalities of the app (see more in the MVP section). Users can browse upcoming events, sign up for events, engage in in-app private messaging, and utilize QR code generation and scanning for seamless connections. Users can also personalize their experience by customizing their profiles, enabling them to connect more effectively with others and build a strong community. Additionally, event hosts have the ability to create and edit events, ensuring effective event management.
```

File: template/2-history.md [hunk 6]
```diff
@@ -2,42 +2,6 @@
...
-
-*What is missing to bridge from PoC to MVP?*
-- there are still some user flows that we designed that are missing from the PoC
-- some important defining features regarding connecting with others still need to be implemented
-- some more user flows have to be thought through and designed to make sure they are the best for the user, such as group chats, in event geo-location, and history of attended events
+Throughout the development of the PoC, our team faced several learning curves, particularly with the Scrum process. Initially, we encountered challenges with managing large pull requests (PRs) and task clarity, which we addressed by refining task definitions on the Scrum board and limiting the PR scope, which also led to fewer dependencies between tasks. Furthermore, the ambitious nature of the app required us to make difficult prioritization decisions due to limited time. Despite these challenges, we successfully laid the groundwork for the MVP, although further development is needed to implement additional user flows and defining features such as group chats, in-event geo-location, and event history tracking to fully bridge from PoC to MVP.
```
Comment (christinehahaa) [inline template/2-history.md]: What is missing to bridge from PoC to MVP?

Suggestion: split into two paragraphs to make it clear we're answering questions.
```suggestion
Throughout the development of the PoC, our team faced several learning curves, particularly with the Scrum process. Initially, we encountered challenges with managing large pull requests (PRs) and task clarity, which we addressed by refining task definitions on the Scrum board and limiting the PR scope, which also led to fewer dependencies between tasks. Furthermore, the ambitious nature of the app required us to make difficult prioritization decisions due to limited time. 

Despite these challenges, we successfully laid the groundwork for the MVP. Further development is needed to implement additional user flows and define more core features that add value to the user, such as group chats, associating new connections to events, defining privacy levels for individual profiles, event photo sharing, in-event geo-location, and event history tracking to fully bridge from PoC to MVP.
```

File: template/4-value_proposition.md [hunk 7]
```diff
@@ -1,18 +1,7 @@
 # The Value Proposition
 
-*Describe the value proposition and argue that it is:*
+Event Radar is a one-stop platform for event discovery, registration, and connection-making. It simplifies networking by enabling instant friend connections through QR code scanning and in-app messaging and streamlines event hosting with efficient ticket scanning and attendee management tools.
```
Comment (christinehahaa) [inline template/4-value_proposition.md]:
```suggestion
Event Radar is a one-stop platform for event discovery, registration, and connection-making. It simplifies networking by enabling instant friend connections through QR code scanning and in-app messaging, as well as streamlines event hosting with efficient ticket scanning and attendee management tools.
```

File: template/4-value_proposition.md [hunk 8]
```diff
@@ -1,18 +1,7 @@
...
 
-*1. Easy to communicate*
-- One-stop platform for event discovery, registration, and connection-making.
-- Simplifies networking by enabling instant friend connections through QR code scanning.
-- Streamlines event hosting with efficient ticket scanning and attendee management tools.
+Our product is defensible as it solves many user pain points in one concise app. The unique QR code-based networking feature promotes genuine, physical networking, setting Event Radar apart from traditional event apps. We offer comprehensive event management tools that cater to both attendees and event hosts, focusing on enhancing post-event connections and creating a sustainable user base.
```
Comment (christinehahaa) [inline template/4-value_proposition.md]: Event management tools sentence seems superficial. Should focus on the pain point more and the fact that our app is different from others due to the live facilitation of connection making. 

```suggestion
Our product is defensible as it solves many user pain points in one concise app. Have you ever forgotten someone awesome you met at an event, and wished you got their contacts? The unique QR code-based networking feature promotes easy, genuine, physical networking, setting Event Radar apart from traditional event apps. When entering a new environment such as moving countries, it can be hard to make new friends with similar interests. Finding interested users to attend events your events can also be difficult. With EventRadar, we offer comprehensive event management tools that cater to both attendees and event hosts, focusing on enhancing post-event connections and creating a sustainable user base. 
```

File: template/4-value_proposition.md [hunk 9]
```diff
@@ -1,18 +1,7 @@
...
-*2. Defensible*
-- Unique QR code-based networking feature that promotes genuine, physical networking, setting it apart from traditional event apps.
-- Offers comprehensive event management tools that cater to both attendees and event hosts.
-- Focuses on enhancing post-event connections, creating a sustainable user base.
-
-*3. Relevant*
-- Meets the growing demand for seamless event experiences local to users in a digital age.
-- Addresses the need for effective networking and relationship-building beyond event attendance.
-- Aligns with trends in social interaction and professional networking, especially in urban and academic environments.
+Finally, Event Radar stays relevant as it meets the growing demand for seamless event experiences local to users in a digital age. It addresses the need for effective networking and relationship-building beyond event attendance, aligning with trends in social interaction and professional networking, especially in urban and academic environments.
```
Comment (christinehahaa) [inline template/4-value_proposition.md]:
```suggestion
Finally, Event Radar stays relevant as it meets the growing demand for seamless event experiences local to users in a digital post-COVID age. It addresses the need for genuine real-life connections, effective networking and relationship-building beyond event attendance, aligning with trends in social interaction and professional networking, especially in urban and academic environments.
```

**Review by christinehahaa — CHANGES_REQUESTED** [review]
Looks awesome overall, so far! Thanks for doing this. 

I just added a few extra suggestions to really beef it up.

File: template/2-history.md (hunk 5 above)
Comment (jgeaso1266) [reply template/2-history.md]: perhaps "browse and sign-up for upcoming events" would be more succint.

File: template/4-value_proposition.md (hunk 8 above)
Comment (jgeaso1266) [reply template/4-value_proposition.md]: Great point, but I would refrain from making the value proposition sound like a pitch or a marketing ploy. But I will reword it to focus on the pain point.

Comment (christinehahaa) [reply template/4-value_proposition.md]: Sounds good, I think referring explicitly to the pain point will be better. Then it can be understood why this idea came about in the first place.

File: template/2-history.md (hunk 5 above)
Comment (christinehahaa) [reply template/2-history.md]: sounds good

File: template/1-overview.md
Comment (jgeaso1266) [reply template/1-overview.md:1]: I don't agree with some of Grammarly's reasoning in removing important words just to make it more succinct, but I will retain any grammar fixes that don't do that.

File: template/4-value_proposition.md [hunk 10]
```diff
@@ -1,18 +1,7 @@
...
+Event Radar is a one-stop event discovery, registration, and connection-making platform. It simplifies networking by enabling instant friend connections through QR code scanning and in-app messaging. It also streamlines event hosting with efficient ticket scanning and attendee management tools.
 
-*1. Easy to communicate*
-- One-stop platform for event discovery, registration, and connection-making.
-- Simplifies networking by enabling instant friend connections through QR code scanning.
-- Streamlines event hosting with efficient ticket scanning and attendee management tools.
+Our product is defensible because it addresses multiple user pain points in one concise app. Event goers can easily discover nearby events by browsing the events on our app. Those who struggle to remember to get contact information from people they meet at an event can simply scan the other user’s QR code. Event hosts can manage their large attendee lists effortlessly using Event Radar’s event management tools. Our comprehensive solution helps build a sustainable user base and competes with more established solutions that do not address all these pain points.
```
Comment (christinehahaa) [inline template/4-value_proposition.md]: Still think the event host part should focus more on finding interested event attendees to come to their events. 
```suggestion
Our product is defensible because it addresses multiple user pain points in one concise app. Event goers can easily discover nearby events by browsing the events on our app. Those who struggle to remember to get contact information from people they meet at an event can simply scan the other user’s QR code. Event hosts can better reach their intended audience and manage their large attendee lists effortlessly using Event Radar’s event management tools. Our comprehensive solution helps build a sustainable user base and competes with more established solutions that do not address all these pain points.
```

File: template/2-history.md [hunk 11]
```diff
@@ -2,42 +2,8 @@
...
-- many members of our team were new to the scrum process, so it was an adjustment period, especially at the beginning, to get used to agile development
-- often at the beginning we were so eager to correct issues with the code that we took on gigantic PRs that touched too many files, we learned from this mistake by 1) limiting the scope of PRs or splitting PRs if needed, and 2) writing better tasks in the SCRUM board that made the scope of the task clear to the developer
-- with an idea as ambitious as ours and a limited time to complete everything, we knew we would not be able to finish everything in time, so we had to make a lot of hard decisions regarding prioritization of tasks
-- we learned to break down tasks in a way that reduces dependencies between them
-
-*What is missing to bridge from PoC to MVP?*
-- there are still some user flows that we designed that are missing from the PoC
-- some important defining features regarding connecting with others still need to be implemented
-- some more user flows have to be thought through and designed to make sure they are the best for the user, such as group chats, in event geo-location, and history of attended events
```
Comment (christinehahaa) [inline template/2-history.md:43]: should this list all features? from PoC to MVP? I think what makes it special is the stuff like, you have to send a photo when u add friend, event photo sharing, privacy levels, etc.

**Review by christinehahaa — CHANGES_REQUESTED** [review]
Some final comments

File: template/2-history.md (hunk 11 above)
Comment (jgeaso1266) [reply template/2-history.md:43]: I guess the question is are all those features part of the MVP (i.e. must these features be present before showing the app to potential customers)?

File: template/4-value_proposition.md (hunk 10 above)
Comment (jgeaso1266) [reply template/4-value_proposition.md]: I would say it's a stretch to say that Event Radar will help hosts reach their intended audience better than, say, social media apps (which we list as a complimentary product in section 3). But I think maybe we can emphasize that hosts can easily market their event on the app. What do you think?

File: template/2-history.md (hunk 11 above)
Comment (christinehahaa) [reply template/2-history.md:43]: Closed, as discussed offline. Features are not needed / referenced in other parts of report.

**Review by christinehahaa — APPROVED** [review]
LGTM!
